WORKFLOW.md

```diff
@@ -0,0 +1,38 @@
+# Workflow
+
+**Note:** As much as possible, this workflow has to be follow when
+  contributing to GreenPix repositories. In exceptional cases,
+  specific workflow can be used and the project README or a dedicated
+  WORKFLOW file will describe it.
+
+Core Team members and regular Contributors (see ROLES.md) have write
+access to GreenPix repositories.  That is, they can push commit
+directly to those repositories. If you are a new contributor, you do
+not have write accesses. You have to fork the targeted repositories
+and push your changes into those forks.
+
+It's a good practice that even Core Team members work on their own fork
+for minor or trivial features and keep branches in upstream
```

Comment:
Yep. Working on fork or directly on upstream does not really change the work flow, that is branching, commiting, PR-ing.

Working with forks has one issue, through. If one wants to work with other contributors, it has to give them the necessary rights. However, it's how Github works, there is little thing we can do about it :).